baybe/telemetry.py

```diff
@@ -72,6 +72,13 @@
 
 Note, however, that (un-)setting the variable in the shell will not affect the running
 Python session.
+
```

Comment:
We now have telemetry information somehow scattered across different files, each containing a different subset of the information. In particular, there's stuff in README.md, stuff in telemetry.py and stuff in the new user guide. I suggest to remove the content from `telemetry.py`, and then only copy the most important sections to README.md, that is, everything that's already there + the new uninstallation note as additional bullet. Plus, link from the readme to the userguide section instead. What do you think?

Reply:
Very nice, thanks 👏🏼 

Regarding link: yes, there is a link, but it points to the telemetry.py module, where you have deleted the instructions just now. So I think this should be redirected to the new env-var section (not the page itself, but really the section)

Reply:
And perhaps we could also keep a link in telemetry.py in the body of the module docstring, somehow explaining that info about the configuration can be found in the new section

Reply:
i see, but changing the link will make the pipeline fail similar to the rephrashing of a user guide renaming

Reply:
updated the links but it comes with the potential issues of linking non existing pages

Reply:
What does it mean for this PR (and the subsequent ones)? That the pipeline will fail all the time? In that case, I'd suggest that we add the last commit rather to #217...

And for the link in `telemetry.py`: I think the way you did it, it's not being link checked and will not appear nicely in the docs. I'd prefer to have an actual "sphinx"-link in there (a la `ref` or whatever keyword is suitable for this context)

Reply:
not sure what you mean bout the formatting
links like that seem to be rendered fine, see eg:
![image](https://github.com/emdgroup/baybe/assets/17951239/2e80a684-920a-43e5-8f49-5c7c5c5a96e1)

Ive dropped the commit form this PR and added the link change to #217

Reply:
I mean that instead of having a plain link a la `https://...` that points to one of our internal doc pages, we should rather use a sphinx ref, which is specifically designed for that. But since you have moved it to 217, this PR is now ready. I can make the corresponding change in the other PR 👍🏼